build.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    if std::env::var("TARGET").unwrap().contains("-apple") {
```

Comment:
```suggestion
    #[cfg(target_vendor = "apple")]
```

The `TARGET` environment variable contains the whole triplet which contains several other pieces of information. I think this would be better to specifically get the information that is needed here.

Reply:
Good idea! Addressed in https://github.com/frewsxcv/rust-dark-light/pull/9/commits/d88be191e6bb0a25e02649a55eea57cff55a8432